src/private/WRLDMapView.mm

```diff
@@ -935,6 +938,27 @@ - (void)collapseIndoorMapView
     }
 }
 
+- (void)setEntityHighlights:(NSString*)indoorId highlightIds:(NSArray<NSString*>*)highlightIds color:(UIColor*) color
```

Comment:
Think this probably needs renaming "setIndoorEntityHighlights", and similar rename for clearEntityHighlights and other api points.
On wrld.js, the corresponding api point is on the Indoors api, so has context that this is only applied to indoor map entities:
https://github.com/wrld3d/wrld.js/blob/009b8456950f58e462c6c00ee1f251f1f738ee7f/src/private/indoors_module.js#L365

For parameter names, think try and be consistent (though these were only recently renamed in C++ api for consistency reasons), see:
https://github.com/wrld3d/eegeo-mobile/blob/7fe959dfbf41346ce3039c2d44c9c59544f3cc24/eegeo-api/EegeoIndoorEntityApi.h#L31
"indoorMapId" rather than "indoorId"
"indoorEntityIds" rather than "highlightIds"